skglm/experimental/sqrt_lasso.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
+from numpy.linalg import norm
+from sklearn.linear_model import Lasso
+
+from skglm.penalties import L1
+from skglm.utils import compiled_clone
+from skglm.datafits.base import BaseDatafit
+from skglm.solvers.prox_newton import prox_newton
+
+
+class SqrtQuadratic(BaseDatafit):
+    """norm(y - Xw) / sqrt(len(y))."""
+
+    def __init__(self):
+        pass
+
+    def get_spec(self):
+        spec = ()
+        return spec
+
+    def params_to_dict(self):
+        return dict()
+
+    def value(self, y, w, Xw):
+        return np.linalg.norm(y - Xw) / np.sqrt(len(y))
+
+    def raw_grad(self, y, Xw):
+        minus_residual = Xw - y
+        return minus_residual / norm(minus_residual) / np.sqrt(len(y))
+
+    def raw_hessian(self, y, Xw):
+        n_samples = len(y)
+        fill_value = 1 / (np.sqrt(n_samples) * norm(y - Xw))
+        return np.full(n_samples, fill_value)
+
+
+class SqrtLasso(Lasso):
+
+    def __init__(self, alpha=1., max_iter=100, max_pn_iter=1000, p0=10,
+                 tol=1e-4, fit_intercept=True, verbose=0,):
+        super(Lasso, self).__init__(
+            alpha=alpha, tol=tol, max_iter=max_iter,
+            fit_intercept=fit_intercept)
```

Comment:
I realize this will be an issue if an intercept is fitted, because (to double check) the centering trick will be performed by sklearn Lasso.fit, but it does not work for the SqrtLasso